lib/rails/generators/channel/templates/application_cable_connection.rb.erb

```diff
@@ -0,0 +1,19 @@
+module ApplicationCable
+  class Connection < ActionCable::Connection::Base
+    identified_by :current_user

```

Comment:
Shouldn't have contents to this class, this will vary from implementation to implementation.